python/phylanx/ast/physl.py

```diff
@@ -227,6 +228,11 @@ def __init__(self, func, tree, kwargs):
         check_return(self.ir)
         self.__src__ = self.generate_physl(self.ir)
 
+        if kwargs["dump_physl"] == True:
+            ofn = "./dump_pyhsl_" + kwargs["python_src_tag"] + ".txt"
```

Comment:
Same here: let's use the named argument to determine where to dump the information to.